pages/api/get-update-activity.ts

```diff
@@ -0,0 +1,25 @@
+import mongoose from "mongoose";
+import { NextApiRequest, NextApiResponse } from "next";
+import { updateModel } from "../../models/models";
+import dbConnect from "../../utils/dbConnect";
+import { cleanForJSON } from "../../utils/utils";
+
+export default async function getCurrUserUpdatesHandler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
i feel like this should be named something clearer so that we know what it does later....

Reply:
also this API route should be named j "activity" or "update-activity", get is in the request method

Reply:
ok hot i was just following the existing convention